test/run_test.py

```diff
@@ -435,20 +435,43 @@ def pull_job_times_from_S3() -> Dict[str, Tuple[float, int]]:
     return calculate_job_times(s3_reports)
 
 
+def get_S3_job_times() -> Dict[str, float]:
```

Comment:
naming is a bit confusing consider we already have `pull_job_times_from_S3`, how about
```suggestion
def get_historic_job_time() -> Dict[str, float]:
```